### PR TITLE
Includes submit button value into FormData

### PR DESCRIPTION
**Pull request by brunobasto**

### CODE REVIEW

**Comment by SpencerWoo** [comment]
Any updates on this?

**Comment by brunobasto** [comment]
Hey @SpencerWoo, the proposed fix is now in review. Thx

File: src/screen/RequestScreen.js [hunk 1]
```diff
@@ -238,6 +235,18 @@ class RequestScreen extends Screen {
 	}
 
 	/**
+	 * Adds aditional data to the body of the request in case a submit button
+	 * is captured during form submission.
+	 * @param {!FormData} body The FormData containing the request body.
+	 */
+	maybeAppendSubmitButtonValue(body) {
+		const button = globals.capturedFormButtonElement;
+		if (body && button && button.name && !button.disabled) {
```
Comment (fernandosouza) [inline src/screen/RequestScreen.js]: Why are you checking if the button is not disabled here? Once the disabled button prevents the form submission the screen will not be loaded. Right?

Comment (brunobasto) [inline src/screen/RequestScreen.js]: I added that as a safety measure to conform with the spec, but you're right, It shouldn't be needed. Let me check with that works across all browsers.

**Comment by eduardolundgren** [comment]
LGTM.

**Comment by SpencerWoo** [comment]
Awesome, thanks @brunobasto!  Thanks guys.

**Pull request merged**